charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -8,7 +8,7 @@ sources:
   - https://github.com/newrelic/nri-kubernetes/tree/master/charts/newrelic-infrastructure
   - https://github.com/newrelic/infrastructure-agent/
 
-version: 3.7.1
+version: 3.7.3
```

Comment:
this could be 3.7.2

Reply:
I overlooked this comment and I merged it with 3.7.3, sorry.